### PR TITLE
Use generics in RunGetQueryForSchema

### PR DESCRIPTION
**Pull request by janisz**
## Description

Look at 
- [ ] `pkg/search/postgres/common.go`
- [ ] `tools/generate-helpers/pg-table-bindings/store.go.tpl`

## Testing Performed

CI

### CODE REVIEW

File: pkg/search/postgres/common.go [hunk 1]
```diff
@@ -735,6 +735,35 @@ func RunGetQueryForSchema(ctx context.Context, schema *walker.Schema, q *v1.Quer
...
+		if err := row.Scan(&data); err != nil {
+			return nil, err
+		}
+		msg := PT(new(T))
+		if err := msg.Unmarshal(data); err != nil {
+			return nil, err
+		}
+
+		return (*T)(msg), nil
+	})
```
Comment (misberner) [inline pkg/search/postgres/common.go:764]:
```suggestion
	data, err := RunGetQueryForSchema(ctx, schema, q, db)
	if err != nil {
		retur nil, err
	}
	msg := PT(new(T))
	if err := msg.Unmarshal(data); err != nil {
		return nil, err
	}
	
	return (*T)(msg), nil
```
to reduce code duplication?

**Review by misberner — CHANGES_REQUESTED** [review]
The slice/`GetMany` variant has a performance advantage. This is really only a cosmetic change, no? I'm not against it, I just doubt how effective this is given that at least in this PR you only change generated code. If this code will also be used in handwritten code, I think it's reasonable.

File: pkg/search/postgres/common.go (hunk 1 above)
Comment (janisz) [reply pkg/search/postgres/common.go]: Good point, I extracted unamrshal and scanRows methods so and fixed handwritten code

**Review by md2119 — APPROVED** [review]

**Comment by janisz** [comment]
Merging as failures are known issue https://issues.redhat.com/browse/ROX-13242

**Pull request merged**